instance/migrations/0130_add_blank_fields_to_appserver_and_deployment.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.12 on 2020-06-29 23:32
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0129_add_deployments_and_appserver_count'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='deployment',
+            name='creator',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to='userprofile.UserProfile'),
+        ),
+        migrations.AlterField(
+            model_name='openedxappserver',
+            name='owner',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='openedxappserver_set', to='instance.InstanceReference'),
+        ),
```

Comment:
It looks like you need to re-generate migrations ; these seem to be outdated as they do not correspond to the changes. I believe that's also why the `quality_migrations_tests` test is currently failing.